src/components/Icons/IconClose.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+export const IconClose: React.FC = () => (
+	<svg
+		width="18"
+		height="18"
+		viewBox="0 0 18 18"
+		fill="none"
+		xmlns="http://www.w3.org/2000/svg"
+	>
+		<path d="M1 1L17 17M17 1L1 17" stroke="white" />
+	</svg>
```

Comment:
This needs to be exported from zUI

Reply:
I've removed this as it's not being used. Resolved.